src/renderers/shaders/ShaderChunk/bsdfs.glsl

```diff
@@ -1,11 +1,19 @@
-float calcLightAttenuation( const in float lightDistance, const in float cutoffDistance, const in float decayExponent ) {
+// convert from intensity (power / solid angle) to irradiance assuming a spherical light falloff
+// this is only valid for spot and point lights
+// physically correct inverse quadratic falloff is achieved when decayExponent is 2.
+// discussed here: https://github.com/mrdoob/three.js/issues/7608
+float intensityToIrradianceSphericalFalloffCoefficient( const in float lightDistance, const in float cutoffDistance, const in float decayExponent ) {
 
 	if ( decayExponent > 0.0 ) {
 
-	  return pow( saturate( -lightDistance / cutoffDistance + 1.0 ), decayExponent );
+		float distanceFalloff = 1.0 / pow( max( lightDistance, EPSILON ), decayExponent );
```

Comment:
With this change, the user's scene is required to have world distance units of meters.

Can we work around this problem by adding `THREE.DistanceUnits` and converting?